src/MCP2515.cpp

```diff
@@ -45,107 +45,88 @@ uint8_t MCP2515::begin(const uint8_t bus_rate) {
 
     res = set_control_mode(Mode::Normal);
     if (res != Result::OK) {
-        #if DEBUG_MODE
+#if DEBUG_MODE
         trace << "Retuning to normal mode failed\n";
-        #endif
+#endif
         return Result::Failed;
     }
 
-    #if DEBUG_MODE
+#if DEBUG_MODE
     trace << "Returned to normal mode\n";
-    #endif
+#endif
     return Result::OK;
 }
 
 uint8_t MCP2515::set_mask(uint8_t mask_number, uint32_t mask) {
 
     uint8_t res = Result::OK;
-    #if DEBUG_MODE
+#if DEBUG_MODE
     trace << "Starting mask setup\n";
-    #endif
+#endif
     res = set_control_mode(Mode::Config);
     if (res > 0) {
-        #if DEBUG_MODE
+#if DEBUG_MODE
         trace << "Switching to config mode failed\n";
-        #endif
+#endif
         return Result::Failed;
     }
 
-    if (mask_number == 0)
+    if (mask_number == 0) {
         write_id(Register::RXM0SIDH, id);
-    else if(mask_number == 1)
+    } else if (mask_number == 1) {
         write_id(Register::RXM1SIDH, id);
-    else
+    } else {
         return Result::Failed;
+    }
 
     res = set_control_mode(Mode::Normal);
     if (res != Result::OK) {
-        #if DEBUG_MODE
+#if DEBUG_MODE
         trace << "Returning to normal model failed\nMask setup failed\n";
-        #endif
+#endif
         return res;
     }
-    #if DEBUG_MODE
+#if DEBUG_MODE
     trace << "Mask setup successfull\n";
-    #endif
+#endif
 
     return Result::OK;
 }
 
 uint8_t MCP2515::set_filter(uint8_t filter_number, uint32_t filter)
 {
```

Comment:
Some places you have braces under the function on a new line while for some you have on the same line. Just make it consistent choose 1. Most of our code bases uses braces on the same line

Reply:
Fixed